### PR TITLE
Varun/rwa5 neareast gap

### PR DESCRIPTION
**Pull request by varunasthana92**
Position the robot to the nearest gap from the part to pick

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,5 +2,9 @@
     "python.autoComplete.extraPaths": [
         "/home/varun/ariac_ws/devel/lib/python2.7/dist-packages",
         "/opt/ros/melodic/lib/python2.7/dist-packages"
-    ]
```
Comment (SaumilShah66) [inline .vscode/settings.json:5]: Remove this files from github

File: rwa5_4/config/rwa5_sensor_kootbally1.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
+sensors:
+
+  breakbeam_0a:
+    type: break_beam
+    pose:
```
Comment (SaumilShah66) [inline rwa5_4/config/rwa5_sensor_kootbally1.yaml]: I guess you mentioned about changing [a,b,c,d,e] to numbers ?! Fine with this also. Just wanted to make sure.

Comment (varunasthana92) [reply rwa5_4/config/rwa5_sensor_kootbally1.yaml]: This was an old reference file, now deleted

File: .vscode/settings.json (hunk 1 above)
Comment (varunasthana92) [reply .vscode/settings.json:5]: Deleted

**Pull request merged**